src/shared/Input.js

```diff
@@ -275,13 +265,13 @@ export default class Input extends Component {
   };
 
   addClass = (e) => {
-    if (this.props.className) {
-      e.target.classList.add(this.props.className);
+    if (this.props.classes) {
+      e.target.classList.add(this.props.classes);
```

Comment:
It should be `className` to get the default `className` of the element. And if didn't use it, remove the file.